cob_mimic/src/mimic_node.py

```diff
@@ -60,6 +60,7 @@
 import sys
 import os
 import subprocess
+import vlc
```

Comment:
Just had a chat with @ipa-nhg we could go the ```rosdep``` way and install vlc python bindings.